### PR TITLE
Example for MongoDB integration with Kanister

### PR DESCRIPTION
**Pull request by PrasadG193**
## Change Overview

In this PR:
- Added blueprint to work with upstream helm chart for backup, restore and delete operations
- Updated README with steps to demonstrate mongodb data protection with Kanister

## Pull request type

Please check the type of change your PR introduces:
- [ ] Work in Progress
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Trival/Minor
- [ ] Bugfix
- [ ] Feature
- [x] Documentation

## Issues

- #XXX

## Test Plan

- [ ] Manual
- [ ] Unit test
- [ ] E2E

### CODE REVIEW

File: docker/mongodb/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,5 @@
+FROM mongo:3.6
```
Comment (tdmanv) [inline docker/mongodb/Dockerfile]: Let's use the most recent version of mongo

Comment (PrasadG193) [reply docker/mongodb/Dockerfile]: Raised separate PR for this: https://github.com/kanisterio/kanister/pull/274

Comment (tdmanv) [reply docker/mongodb/Dockerfile]: [minor] We don't need to copy the license in this directory. we can just include the copyright header.

Comment (PrasadG193) [reply docker/mongodb/Dockerfile]: I am not sure if this PR is valid anymore because of #274

**Comment by tdmanv** [comment]
@PrasadG193 should we still merge this one?

**Comment by PrasadG193** [comment]
@tdmanv We have already merged https://github.com/kanisterio/kanister/pull/274 which uses the latest version of MongoDB. This is no longer valid. Hence closing the PR

**Pull request closed without merging**